src/System.Windows.Forms/src/System/Windows/Forms/Controls/PropertyGrid/PropertyGrid.cs

```diff
@@ -1684,7 +1684,7 @@ private PropertyTab CreateTab(Type tabType, IDesignerHost host)
                 else
                 {
                     // Just call the default constructor.
-                    tab = (PropertyTab)Activator.CreateInstance(tabType);
+                    tab = (PropertyTab)Activator.CreateInstance(tabType)!;
```

Comment:
Would this be better since `tab` is type `PropertyTab?`
```suggestion
                    tab = (PropertyTab?)Activator.CreateInstance(tabType);
```

Reply:
We have used null-forgiving operator with `Activator.CreateInstance` in most of our codebase.